src/planscape/collaboration/models.py

```diff
@@ -0,0 +1,74 @@
+from django.contrib.gis.db import models
+from django.contrib.contenttypes.models import ContentType
+from django.utils.translation import gettext_lazy as _
+from django.contrib.auth import get_user_model
+import uuid
+
+User = get_user_model()
+
+
+class Role(models.TextChoices):
+    CREATOR = "Creator"
+    OWNER = "Owner"
+    COLLABORATOR = "Collaborator"
+    VIEWER = "Viewer"
+
+
+class Permissions(models.Model):
+    role = models.CharField(
+        choices=Role.choices,
+        max_length=16,
+        default=Role.VIEWER,
+    )
+    permission = models.CharField(max_length=60)
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=["role", "permission"],
+                name="unique_permission",
+            )
+        ]
+
+
+class Collaborator(models.Model):
+    uuid = models.UUIDField(default=uuid.uuid4, unique=True)
```

Comment:
Is this just meant to be a unique key, or is there another use for this UUID?

Reply:
unique id, do i need to add this manually or does django add it by default?

Reply:
Yes, Django adds this automatically, unless you tell it otherwise, basically:
https://docs.djangoproject.com/en/4.1/topics/db/models/#automatic-primary-key-fields
It's a little hard to find, but it's already added this to your migration file, above.